.pre-commit-config.yaml

```diff
@@ -26,7 +26,6 @@ repos:
     - id: trailing-whitespace
     - id: end-of-file-fixer
     - id: no-commit-to-branch
-      args: [--branch, main]
```

Comment:
why this change?

Reply:
Because `no-commit-to-branch` now default to branches named "master" _and_ "main". Not necessary at all to this PR though